packages/ng/button/button.component.ts

```diff
@@ -0,0 +1,53 @@
+import { booleanAttribute, ChangeDetectionStrategy, Component, inject, Input, OnChanges } from '@angular/core';
+import { NgClazz, Palette } from '@lucca-front/ng/core';
+
+@Component({
+	// eslint-disable-next-line @angular-eslint/component-selector
+	selector: 'button[luButton],a[luButton]',
+	standalone: true,
+	hostDirectives: [NgClazz],
+	template: '<ng-content></ng-content>',
+	styleUrls: ['./button.component.scss'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	host: {
+		class: 'button',
+	},
+})
+export class ButtonComponent implements OnChanges {
+	#ngClazz = inject(NgClazz);
+
+	@Input()
+	size: 'M' | 'S' | 'XS' = 'M';
```

Comment:
Hm, I don't think so for now. Because the only use case would be to have an input in another component that can give a size to the button that's different from its ownsize and I don't see this happening anytime soon.